sockets/utils/getSocketUrlParts.js

```diff
@@ -82,6 +82,11 @@ function getSocketUrlParts(resourceQuery) {
   pathname = parsedQuery.sockPath || pathname;
   port = parsedQuery.sockPort || port;
 
+  // Make sure the protocol from resource query has a trailing colon
+  if (parsedQuery.sockProtocol) {
+    protocol = url.format({ protocol: parsedQuery.sockProtocol });
```

Comment:
Hmm. Should we just manually format the protocol ourselves?

Technically speaking it is safe to just to add a `:`, since we any protocol received from the Webpack options that does not conform the the `enum` will throw?